train_monomer_demo.sh

```diff
@@ -2,15 +2,16 @@
 [ -z "${n_gpu}" ] && n_gpu=$(nvidia-smi -L | wc -l)
 export NCCL_ASYNC_ERROR_HANDLING=1
 export OMP_NUM_THREADS=1
-mkdir -p $1
+mkdir -p temp
```

Comment:
once this proposal is validated, modifications to this file can be dropped; was just for easy testing of the code